src/app.rs

```diff
@@ -227,7 +227,8 @@ impl<AppData: Send + Sync + 'static> App<AppData> {
             .ok_or(std::io::ErrorKind::InvalidInput)?;
 
         println!("Server is listening on: http://{}", addr);
-        http_service_hyper::serve(self.into_http_service(), addr);
+        // TODO: Use this future somehow.
+        let _ = http_service_hyper::serve(self.into_http_service(), addr);
```

Comment:
I had to suppress the `unused_must_use` here because of `deny(warnings)`. I assume there's a plan to pass this future out, or turn this into an async function or something, that can happen after this is merged?

Reply:
It seems like you're encountering changes landed in [rustasync/http-service#21](https://github.com/rustasync/http-service/pull/21), which is related to #163. Having `serve` return a `Future` is part of our ongoing process to make initializing Tide slightly easier.

Reply:
Yeah, `http-service-hyper` had a breaking change that renamed `serve` to `run`. Changing the function call should do the thing!

Reply:
Ah, cool, I've updated this to use `run` for now then.